install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,124 +11,248 @@
   nfs: <4>
-    path: "/tmp" <5>
-    server: "172.17.0.2" <6>
-  persistentVolumeReclaimPolicy: "Recycle" <7>
-----
-<1> The name of the volume. This will be how it is identified via
-link:../../architecture/additional_concepts/storage.html[persistent volume
-claims] or from pods.
-<2> The amount of storage allocated to this volume.
-<3> This is the policy for restricting read/writes to a shared volume.
-<4> This defines the volume type being used, in this case the *nfs* plug-in.
-<5> The path that is exported by the NFS server.
-<6> The host name or IP address of the NFS server.
-<7> Defines what happens to a volume when released from its claim. Valid options
+    path: /tmp <5>
+    server: 172.17.0.2 <6>
+  persistentVolumeReclaimPolicy: Recycle <7>
+```
+<1> the name of the volume. This is the PV identify in various `oc ... pod` commands.
+<2> the amount of storage allocated to this volume.
+<3> though this appears to be related to controlling access to the volume, in fact, it
+is actually used similarly to labels and used to match a PVC to a PV. Currently, no
+access rules are enforced based on the `accessModes`.
+<4> this defines the volume type being used, in this case the *nfs* plug-in.
+<5> the path that is exported by the NFS server.
+<6> the host name or IP address of the NFS server.
+<7> defines what happens to a volume when released from its claim. Valid options
 are *Retain* (default) and *Recycle*. See
 link:#nfs-reclaiming-resources[Reclaiming Resources].
-====
 
-Save your definition to a file, for example *_nfs-pv.yaml_*, and create the
-persistent volume:
-
-====
-----
+Save the definition to a file, for example *_nfs-pv.yaml_*, and create the persistent
+volume:
+```
 # oc create -f nfs-pv.yaml
 persistentvolume "pv0001" created
-----
-====
+```
 
 Verify that the persistent volume was created:
-
-====
-----
+```
 # oc get pv
 NAME                     LABELS    CAPACITY     ACCESSMODES   STATUS      CLAIM     REASON    AGE
 pv0001                   <none>    5368709120   RWO           Available                       31s
-----
-====
+```
 
-Users can then link:../../dev_guide/persistent_volumes.html[request storage
-using persistent volume claims], which can now utilize your new persistent
-volume.
+The next step can be to create a persistent volume claim (PVC) which will bind to the new PV.
+This is easy to do, as shown in the "nfs-claim.yaml example below:
+```
+#claim yaml file: "nfs-claim.yaml"
 
-[[nfs-enforcing-disk-quotas]]
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-claim1
+spec:
+  accessModes:
+    - ReadWriteOnce <1>
+  resources:
+    requests:
+      storage: 1Gi <2>
+
+# oc create -f nfs-claim.yaml
+```
+<1> as mentioned above for PVs, the `accessModes` do not enforce security, but rather act as
+labels to match a PV to a PVC.
+<2> this claim will look for PVs offering 1Gi or greater capacity.
 
+*Note:* PVs, and PVCs are not necessary, just convenient, and make sharing a volume across a
```

Comment:
"Use a [NOTE] block" applies to the rest of the `*Note:*`s below as well.

Drop the comma after "PVs".